README.md

```diff
@@ -100,6 +100,8 @@ An optional storage caching CLI flag `--routing.cache-targets` can be leveraged
 
 To the see list of available metrics, run `./bin/eigenda-proxy doc metrics`
 
+To quickly set up monitoring dashboard, add eigenda-proxy metrics endpoint to a reachable prometheus server config as a scrape target, add prometheus datasource to Grafana to, and import the existing [Grafana dashboard JSON file](./grafana_dashboard.json)
+
 ## Deployment Guide
```

Comment:
if we're gonna provide an example dashboard should we also add a docker compose that allows users to optionally stand up observability?

Reply:
I added a docker compose with eigenda-proxy, prometheus, and grafana!
was thinking to also add a bash script service that just send some test traffic to the running eigenda-proxy, but had some trouble with building and executing it 🤔 (lost 2 hours on this:( lmk if you think it's worth spending more time on)

Reply:
eh I'd just let it be :)